.circleci/config.yml

```diff
@@ -136,6 +136,37 @@ jobs:
               --branch $CIRCLE_BRANCH \
               --message "$GIT_COMMIT_DESCRIPTION"
 
+  integration_tests_with_parity:
+    docker:
+      - image: paulperegud/elixir-omg-circleci:v1.8-parity
+        environment:
+          DATABASE_URL: postgres://postgres:postgres@localhost:5432/omisego_dev
+      - image: circleci/postgres:9.6-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: omisego_dev
+          MIX_ENV: test
+    working_directory: ~/repo
+
+    steps:
+      - attach_workspace:
+           at: .
+
+      - run: mix local.hex --force
+      - run: mix local.rebar --force
+
+      - run:
+          name: Integration Tests with Parity
+          environment:
+            SIGNER_PASSPHRASE: ThisIsATestnetPassphrase
```

Comment:
Just as a heads up: I know this has been configured for testing but you can set this to be populated from a secret in CircleCI.

Reply:
Because of an issue integration tests (database sandbox fail or it crashes and we write skipping it) I will not add parity integration test for CI for now.